site/en/blog/anonymous-iframe-origin-trial/index.md

```diff
@@ -0,0 +1,159 @@
+[`performance.measureUserAgentSpecificMemory()`](https://web.dev/monitor-total-page-memory-usage/),
+and
+[high-precision timers with better resolution](/blog/cross-origin-isolated-hr-timers/).
+
+To enable cross-origin isolation, websites must send the following two HTTP
+headers:
+
+```text
+Cross-Origin-Embedder-Policy: require-corp
+Cross-Origin-Opener-Policy: same-origin
+```
+{% Aside %}
+[`COEP:credentialless`](/blog/coep-credentialless-origin-trial/#credentialless-to-the-rescue) can also be used as an alternative to `require-corp`.
+{% endAside %}
+
+## Challenges with enabling COEP
+
+While cross-origin isolation brings webpages better security and the ability to
+enable powerful features, deploying COEP can be
+[difficult](https://web.dev/cross-origin-isolation-guide/). One of the biggest
+challenges is that all cross-origin iframes must also deploy COEP and CORP.
+Iframes without those headers will not be loaded by the browser.
+
+The iframes are usually served by a third party for whom it may not be easy to
+deploy COEP.
+
+## Anonymous iframe to the rescue
+
+That's where anonymous iframe comes in. By adding the `anonymous` attribute to
+the `<iframe>` element, the iframe is loaded from a different, ephemeral storage
+partition and it isn't subject to COEP restrictions anymore.
+
+Example:
+
+```html
+<iframe anonymous src="https://example.com">
+```
+
+Iframe is created in a new ephemeral context and doesn't have access to any of
+the cookies associated with the top level website. It starts from an empty
+cookie jar. Likewise, storage APIs such as
+[`LocalStorage`](https://developer.mozilla.org/docs/Web/API/Window/localStorage),
+[`CacheStorage`](https://developer.mozilla.org/docs/Web/API/CacheStorage),
+[`IndexedDB`](https://developer.mozilla.org/docs/Web/API/IndexedDB_API), and so
+on, are loading and storing data in the new ephemeral partition. The partition
+is scoped to the current top-level document and origin of the iframe. Storage
+will be cleared once the top-level document is unloaded.
+
+Anonymous iframes are not subject to COEP embedding rules. This is still
+secure, because they are loaded from a new empty context everytime. They will be
+loaded without their data being personalized. They contain only public data,
+which is not valuable to an attacker.
+
+## Demo
+
+You can check out an anonymous iframe at:
+[https://anonymous-iframe.glitch.me/](https://anonymous-iframe.glitch.me/)
+
+## Register for an origin trial
+
+To ensure that Anonymous iframes are helping developers adopt cross origin
+isolation, we are making them available in Chrome from version 106 to 108 as an
+origin trial.
+
+Register for the origin trial to enable your website to use Anonymous
+iframes:
+
+1.  [Request a token](/origintrials/#/view_trial/537054255563931649)
+    for your origin.
+1.  Use the token in one of the following ways:
+    -  In your HTML:
+
+      `<meta http-equiv="Origin-Trial" content="TOKEN_GOES_HERE">`
```

Comment:
Can't we remove this empty line + 2 spaces to align the sample code?

```suggestion
1.  Use the token in one of the following ways:
    -  In your HTML:  
      `<meta http-equiv="Origin-Trial" content="TOKEN_GOES_HERE">`
```

Reply:
Fixed, check the preview https://deploy-preview-3611--developer-chrome-com.netlify.app/blog/anonymous-iframe-origin-trial/